typings.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "angular2-google-maps",
   "globalDevDependencies": {
-    "es6-shim": "github:DefinitelyTyped/DefinitelyTyped/es6-shim/es6-shim.d.ts#4de74cb527395c13ba20b438c3a7a419ad931f1c",
```

Comment:
I switched to use core-js instead, as it's used by majority of Angular 2 RCx projects now (instead of es6-shim & reflect-metadata). In another words core-js contains what's on es6-shim & reflect-metadata.